spec/dish_spec.rb

```diff
@@ -0,0 +1,20 @@
+
+require 'dish'
+
+describe Dish do
+  context 'adding a dish' do
+    it 'has a #name' do 
+      expect(subject.name).to eq("Hash brown")
+    end
+
+    it 'has a #price' do 
+      expect(subject.price).to eq(0.99)
+    end
+  end
+
+  context 'menu format' do
+    it 'combines name and price in a hash' do 
+      expect(subject.itemise).to eq({ "Hash brown" => 0.99 })
```

Comment:
I'd also try a similar test but using custom values from the name and price, rather than just default ones (to catch potential bugs where default values are not overwritten with given ones - that's neatpicking though!)